json/src/test_helpers/skip.rs

```diff
@@ -24,6 +24,10 @@ pub struct SkipTests {
 	pub block: Vec<SkipBlockchainTest>,
 	/// State tests
 	pub state: Vec<SkipStateTest>,
+	/// Legacy block tests
```

Comment:
We need to document what we mean by "legacy" here. Are these tests that are present in the upstream repository but not runnable/applicable in today's ethereum? Should we provide an explicit date?

Reply:
`Legacy` there is about a folder that exists in the current ethereum/tests repo, where all tests `<Istablul` are moved if I recall correctly. I think that is important to still run those tests, to check if there's any regression in blocks `<Istambul`.

Reply:
Cool, so maybe something like "Legacy tests are tests relevant to previous hardforks and replaced as time goes on; kept here to guard against regressions."?

Reply:
Finally I put "Legacy tests, still keeping it to check if there is any regression in blocks < Instambul HF", more or less the same...